certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      },
+      "post": {
+        "operationId": "PostEnvelopesEnvelopeIdRecipients",
+        "summary": "Add recipient to envelope",
+        "description": "Adds a recipient to a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/RecipientInput"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {},
+            "headers": {
+              "Location": {
+                "description": "Location",
+                "type": "string",
+                "x-ms-summary": "Location"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/envelopes/{envelope-id}/recipients/{recipient-id}": {
+      "get": {
+        "operationId": "GetEnvelopesEnvelopeIdRecipientsRecipientId",
+        "summary": "Get specific recipient for an envelope",
+        "description": "Gets a specific recipient for a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/RecipientId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Recipient"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      },
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeIdRecipientsRecipientId",
+        "summary": "Update recipient by id",
+        "description": "Updates a recipient",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/RecipientId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/RecipientInput"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/put-envelopes-envelope-id-recipients-recipient-id